docs/UserGuide.md

```diff
@@ -200,6 +200,12 @@ Invalid usages
 </div>
 {% endraw %}
 
+<div markdown="span" class="alert alert-primary">:pushpin: **Note:**
+
+The student to be added cannot have the same name and phone number as an existing student, but having either one field the same is alright.
```

Comment:
Could this be clearer?

Maybe adding 3 examples

John Doe 87654321 in list

John Doe 87654321 not allowed

John Doe 98765432 allowed

Jane Doe 87654321 allowed

Reply:
I think this part can change to just "The new student to be added cannot have the same name and phone number as an existing student..."

because below this, there are examples, so technically the second part not really needed I feel